QuickDeployment/QuickDeploymentEnvironment.md

```diff
@@ -2,100 +2,112 @@
 
-> :warning: **WARNING**
->
-> This solution targets environments up to about **1000 nodes**.
-> If you need a solution for a larger environment, QDE would still be suitable as a proof-of-concept, however, best practices would recommend a distributed infrastructure, separating each component into its own discrete node.
-> If you find yourself in need of a more scalable solution, please contact Support and we'll be more than happy to provide guidance for larger solutions.
+---
 
-___
 ## Getting QDE
-To get QDE into your environment, please [reach out to us](https://chocolatey.org/contact/quick-deployment) and we can work to get you set up.
 
-> :memo: **NOTE**: A QDE environment is a fully functional Chocolatey for Business (C4B) environment, as such it will require a business or trial license.
+To get QDE into your environment, please [reach out to us](https://chocolatey.org/contact/quick-deployment) so we can work to get you set up.
 
-___
-## Links
+> :memo: **Note**
+>
+> A QDE environment is a fully functional Chocolatey for Business (C4B) environment; as such, it will require a business or trial license.
 
-* [[QDE Setup|QuickDeploymentSetup]]
-* [[QDE Desktop ReadMe File|QuickDeploymentDesktopReadme]] (included here for convenience)
-* [[QDE SSL/TLS Setup|QuickDeploymentSslSetup]]
-* [[QDE Firewall Changes|QuickDeploymentFirewallChanges]]
-* [[QDE Client Setup|QuickDeploymentClientSetup]] (setting up your client machines)
-* Upgrading? See [How do I upgrade QDE?](#how-do-i-upgrade-qde).
+---
 
-___
 ## Related Articles
 
-A lot of what is done in QDE compresses the work or completely removes the work found in these related articles.
+A lot of what is done in QDE condenses or completely eliminates the work found in these related articles:
 
 * [[Organizational Deployment Guide|How-To-Setup-Offline-Installation]]
 * [Chocolatey Installation](https://chocolatey.org/install#organization)
 * [[Chocolatey Commercial Installation|Installation-Licensed]]
 * [[Automate Package Internalization|How-To-Setup-Internal-Package-Repository]]
 
-> :memo: **NOTE**: If you find that QDE is only good for a proof-of-concept in your environment due to having thousands of endpoints, you will want to understand how to scale out that infrastructure. The above articles address how that is done.
+If you find that QDE is only good for a proof-of-concept in your environment due to having thousands of endpoints, you will want to understand how to scale out that infrastructure.
+The above articles document how that is done.
+
+---
+
+## Links
+
+* [[QDE Setup|QuickDeploymentSetup]]
+* [[QDE Desktop ReadMe File|QuickDeploymentDesktopReadme]] (included here for convenience)
+* [[QDE SSL/TLS Setup|QuickDeploymentSslSetup]]
+* [[QDE Firewall Changes|QuickDeploymentFirewallChanges]]
+* [[QDE Client Setup|QuickDeploymentClientSetup]] (setting up your client machines)
+* [How do I upgrade QDE?](#how-do-i-upgrade-qde).
```

Comment:
Note to @vexx32 - do we want internet enablement link here or down in v1 docs? Something for a future change, not part of this one.